protocols/ocean/pipeline.go

```diff
@@ -60,62 +60,67 @@ func PipelineAll(log *log.Logger) (*PipelineResult, error) {
 			orderCount
 		  }}`
 
-	pages, err := paginatedGraphQuery(baseDatatokensQuery, new(DatatokenResponsePage))
-	if err != nil {
-		log.Println("Error connecting to GraphQL", err)
-		return nil, err
-	}
+	// We have Pools and Datatokens, so we can now construct Assets. A Datatoken
+	// may not have a Pool, or even a DDO associated with it. But we're only
+	// interested in Datatokens with Pools and DDOs.
+	assets := make([]*Asset, 0)
+	addresses := make([]*Address, 0)
 
-	var dtr []DatatokenResponse
-	for _, p := range pages {
-		page := p.(*DatatokenResponsePage)
-		dtr = append(dtr, page.Flatten()...)
-	}
+	for _, config := range oceanScanConfigs {
 
-	var datatokens []*Datatoken
-	for _, datatokenResponse := range dtr {
-		dt, err := NewDataTokenFromDatatokenResponse(datatokenResponse)
+		pages, err := paginatedGraphQuery(baseDatatokensQuery, config.SubgraphURL, new(DatatokenResponsePage))
```

Comment:
shouldn't we update the command to query each subgraph for the different networks?

Reply:
on line 69 we iterate over `oceanScanConfigs` which is a slice of subgraph URLs for every network.
in this case, upon every iteration `config.SubgraphURL` will have a different subgraph URL dedicated to a particular network.

Reply:
Aah ok, true. I can see that